misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/SuspendedContext.java

```diff
@@ -31,22 +31,26 @@
 import org.ballerinalang.debugadapter.utils.PackageUtils;
 
 import java.nio.file.Path;
+import java.util.Map;
 import java.util.Optional;
 
+import static org.ballerinalang.debugadapter.DebugSourceType.DEPENDENCY;
 import static org.ballerinalang.debugadapter.DebugSourceType.PACKAGE;
 import static org.ballerinalang.debugadapter.DebugSourceType.SINGLE_FILE;
+import static org.ballerinalang.debugadapter.utils.PackageUtils.computeProjectKindAndRoot;
 import static org.ballerinalang.debugadapter.utils.PackageUtils.getFileNameFrom;
 
 /**
  * Context holder for debug suspended state related information.
  */
 public class SuspendedContext {
 
+    private final ExecutionContext executionContext;
```

Comment:
Refer comment [here](https://github.com/ballerina-platform/ballerina-lang/pull/30249/files#r622845304)

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/30249/commits/b757b9606646c2a17a343cbd0fc80347236265a6